build/ci_utils/src/cache/goodie/graalvm.rs

```diff
@@ -88,36 +99,37 @@ impl GraalVM {
     pub fn url(&self) -> BoxFuture<'static, Result<Url>> {
         let platform_string = self.platform_string();
         let graal_version = self.graal_version.clone();
+        let graal_version_tag =
+            format!("{}.{}.{}", graal_version.major, graal_version.minor, graal_version.patch);
```

Comment:
As above.

Reply:
Fixed in 57c37ed3f08d7c0b07a14ba9aa6ccaeb8b07c0c5